requirements.txt

```diff
@@ -5,7 +5,7 @@ numpy
 python-dateutil<3.0.0  # denpendency for botocore
 gensim>=0.12.3  # LDA's show topics unified in 0.12.3
 setuptools-git
-Orange3 >=3.25.0
+Orange3 >=3.28.0
```

Comment:
To reviewer: this change is not 100% necessary - I could write tests another way but I noticed that https://github.com/biolab/orange3-text/pull/637 is already rising to 3.28.0 so I used this situation.